Library/Homebrew/manpages/brew.1.md

```diff
@@ -355,6 +355,18 @@ Note that these flags should only appear after a command.
   * `tap --repair`:
     Migrate tapped formulae from symlink-based to directory-based structure.
 
+  * `tap-info` <tap>:
+    Display information about <tap>.
+
+  * `tap-info --json=<version>` (--all|<taps>):
+    Print a JSON representation of <taps>. Currently the only accepted value
+    for <version> is `v1`.
+
+    Pass `--all` to get information on all taps.
```

Comment:
Maybe `--tapped` and `all tapped taps` (or use `installed` instead of `tapped`)?